AsyncDisplayKit/ASCollectionNode.h

```diff
@@ -75,6 +75,20 @@ NS_ASSUME_NONNULL_BEGIN
 @property (weak, nonatomic) id <ASCollectionDataSource> dataSource;
 
 /**
+ * A Boolean value that indicates whether users can select items in the collection node.
+ * If the value of this property is YES (the default), users can select items. If you want more fine-grained control over the selection of items, you must provide a delegate object and implement the appropriate methods of the UICollectionNodeDelegate protocol.
+ */
+@property (nonatomic) BOOL allowsSelection;
```

Comment:
All done, I took the liberty to change the order to two other properties to keep the consistency with the rest of the framework.